src/proctored-exam-settings/ProctoredExamSettings.jsx

```diff
@@ -407,7 +409,13 @@ function ExamSettings(props) {
             const isProctortrack = proctoredExamSettings.proctoring_provider === 'proctortrack';
             setShowProctortrackEscalationEmail(isProctortrack);
             setAvailableProctoringProviders(response.data.available_proctoring_providers);
-            setProctortrackEscalationEmail(proctoredExamSettings.proctoring_escalation_email);
+
+            // The backend API may return null for the proctoringEscalationEmail value, which is the default.
+            // In order to keep our email input component controlled, we use the empty string as the default
+            // and perform this conversion during GETs and POSTs.
+            const proctoringEscalationEmail = proctoredExamSettings.proctoring_escalation_email;
+            setProctortrackEscalationEmail(proctoringEscalationEmail === null ? '' : proctoringEscalationEmail);
```

Comment:
Nit: Is there a reason to be this specific that only `null` gets translated into the empty string? Or would a falseyness check suffice?

Reply:
I wrote it this way because the only valid values for this field on the Studio side are null (the default case) and a non-empty string.